cost-model/src/cost_tracker.rs

```diff
@@ -426,9 +455,10 @@ mod tests {
         let simple_transaction = SanitizedTransaction::from_transaction_for_tests(
             system_transaction::transfer(mint_keypair, &keypair.pubkey(), 2, *start_hash),
         );
-        let mut tx_cost = UsageCostDetails::new_with_capacity(1);
-        tx_cost.programs_execution_cost = 5;
-        tx_cost.writable_accounts.push(mint_keypair.pubkey());
```

Comment:
This was incorrect. Lead to `test_cost_tracker_ok_add_two_same_accounts` failing when we used proper writable accounts count.